.packit.yaml

```diff
@@ -91,6 +91,21 @@ jobs:
              - size: ">= 20 GB"
              - size: ">= 20 GB"
 
+  - job: tests
+    trigger: pull_request
+    identifier: e2e-sanity
```

Comment:
I think you can change identifier to qm-tier-0

Reply:
done